### PR TITLE
S03 us20 super admin home page

### PR DESCRIPTION
**Pull request by RedPandore**

### CODE REVIEW

**Review by bouboumael — DISMISSED** [review]

**Review by bouboumael — DISMISSED** [review]

File: templates/admin/index.html.twig [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        <div class="page-title">
+            <h1 class="h1">Administration</h1>
+        </div>
+        <div class="container-fluid admin-container">
+            <div class="row align-items-center w-100 card-row">
+
+                <div class="col-md-5 col-lg-3 p-2 mx-auto">
+                    <div class="card h-100">
+                        <div class="card-body">
+                            <h5 class="card-title d-flex justify-content-center">
```
Comment (LucyMerlier) [inline templates/admin/index.html.twig]: so we jump from h1 straight to h5?

File: src/Controller/AdminController.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+     */
+class AdminController extends AbstractController
+{
+    /**
+     * @Route("/", name="index")
+     */
+    public function index(): Response
+    {
+        return $this->render('admin/index.html.twig', [
+            'controller_name' => 'AdminController',
```
Comment (LucyMerlier) [inline src/Controller/AdminController.php]: i think you forgot to delete that :slightly_smiling_face:

File: templates/admin/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                </div>
+            </div>
+            <div class="row align-items-center w-100 card-row">
+                <div class="col-md-5 col-lg-3 p-2 mx-auto">
+                    <div class="card h-100">
+                        <div class="card-body">
+                            <h5 class="card-title d-flex justify-content-center">
+                                <i class="fas fa-envelope-open-text logoAdmin"></i>
+                            </h5>
+                            <p class="card-text description">Envoyer des mails a tous les membres inscrit.</p>
```
Comment (LucyMerlier) [inline templates/admin/index.html.twig]: "inscrits" => plural

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: templates/admin/index.html.twig (hunk 3 above)
Comment (RedPandore) [reply templates/admin/index.html.twig]: done

File: src/Controller/AdminController.php (hunk 2 above)
Comment (RedPandore) [reply src/Controller/AdminController.php]: done

File: templates/admin/index.html.twig (hunk 1 above)
Comment (RedPandore) [reply templates/admin/index.html.twig]: done

**Review by bouboumael — DISMISSED** [review]

File: templates/admin/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,96 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}JobPermut Admin
```
Comment (LucyMerlier) [inline templates/admin/index.html.twig]: didn't you do a fix changing "JobPermut" to "Jobpermut"? (with lowercase "p")

**Review by LucyMerlier — COMMENTED** [review]
honest question

File: templates/admin/index.html.twig (hunk 4 above)
Comment (RedPandore) [reply templates/admin/index.html.twig]: Yes i've do this fix, but I forgot this. Sorry

**Review by wonecode — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**